packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -159,23 +151,25 @@ contract ColosseumTest is Colosseum_Initializer {
         address _sender,
         uint8 _turn,
         uint256 _segStart,
-        uint256 _segSize
+        uint256 _segSize,
+        bool _isZkVm
```

Comment:
I've also considered it, but since the functionality to test is proving fault with zkVM proof, this part is not affected by validator system. So I thought it's enough to add a test to either validator V1 or V2 test.